src/image_utils.py

```diff
@@ -5,6 +5,9 @@
 import matplotlib.pyplot as plt
 from ipywidgets import interact
 import math
+from keras_cv import losses
+
+os.environ['TF_ENABLE_ONEDNN_OPTS'] = '0'
```

Comment:
Any reason for disabling this? It usually boosts performance on CPU a fair bit

Reply:
Oh my bad I forgot to remove that.